corehq/motech/fhir/tasks.py

```diff
@@ -0,0 +1,45 @@
+from celery.schedules import crontab
+from celery.task import periodic_task
+
+from corehq import toggles
+from corehq.motech.requests import Requests
+
+from .const import IMPORT_FREQUENCY_DAILY
+from .models import FHIRImporter, FHIRImporterResourceType
+
+
+@periodic_task(run_every=crontab(hour=5, minute=5), queue='background_queue')
```

Comment:
I'm never sure what the right queue to use is, but would the periodic queue be appropriate here?

Reply:
:+1: That sounds right